build.gradle

```diff
@@ -21,7 +21,7 @@ buildscript {
 ext['groovy.version'] = '3.0.0'
 
 group = 'com.synopsys.integration'
-version = '7.10.0-SIGQA3-SNAPSHOT'
+version = '7.10.0-SIGQA4-SNAPSHOT'
```

Comment:
Was QA running from this branch? If so this change should be merged with `master` outside the PR so that if a QA build run from there doesn't share the same version.

Reply:
Good point, should I just modify master's build.gradle version to 7.10.0-SIGQA4-SNAPSHOT?

Reply:
You should cherry-pick the commit made by the build system to `master`
But after the team Slack discussion this morning, I don't think we will be building off branches again.